src/DetailsView/details-view-body.scss

```diff
@@ -54,7 +54,12 @@
 
         :global(.ms-MessageBar-icon) i {
             margin-left: 12px;
-            color: $secondary-text;
+            color: $always-black;
```

Comment:
This is an improvement with Accessibility Insights' in-app high contrast setting, but unfortunately it introduces a new issue with native OS forced color settings. For example, here is what I see with Windows 11's "Aquatic" contrast theme:

![screenshot of message bar with black icon on dark yellow background](https://user-images.githubusercontent.com/376284/180297035-2eb78162-ae7b-4ca7-a217-08e045e59b95.png)

I'd recommend something like this to resolve it for all cases:

```suggestion
            color: $always-black;
            @include ms-high-contrast-override {
                color: inherit;
            }
```

Reply:
ah, good point. didn't test with actual windows high contrast (on Win 10) itself